src/proposal.mligo

```diff
@@ -97,19 +120,15 @@ let stake_tk(token_amount, addr, period, store : nat * address * period * storag
   in { store with freeze_history = new_freeze_history; quorum_threshold_at_cycle = {store.quorum_threshold_at_cycle with staked = new_cycle_staked } }
 
 [@inline]
-let unfreeze_on_ledger (tokens, addr, ledger, total_supply, frozen_token_id, governance_token : nat * address * ledger * total_supply * token_id * governance_token): (operation * ledger * total_supply) =
+let unfreeze_on_ledger (tokens, addr, governance_token : nat * address * governance_token): operation =
```

Comment:
Same two suggestions here, except that I would name this `release_governance_tokens`, or maybe `unlock_governance_tokens`.